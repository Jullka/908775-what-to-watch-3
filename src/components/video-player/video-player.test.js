import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const Movie = {
  poster: `img/bohemian-rhapsody.jpg`,
  video: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

it(`VideoPlayer is rendered correctly`, () => {
  const tree = renderer
   .create(<VideoPlayer
     isPlaying={false}
     poster={Movie.poster}
     video={Movie.video}
   />, {
     createNodeMock: () => ({play: () => {}})
   }).toJSON();

  expect(tree).toMatchSnapshot();
});
