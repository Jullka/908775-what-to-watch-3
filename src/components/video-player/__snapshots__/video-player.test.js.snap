// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is rendered correctly 1`] = `
<div
  className="small-movie-card__image"
>
  <video
    width="100%"
  />
</div>
`;
